index.html

```diff
@@ -1,56 +1,220 @@
-                 feature or website Look through some of my work and 
-                 experience! If you like what you see and have a project
-                  you need coded, don’t hestiate to contact me.</p>
+                <p class="my-profile">I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and
+                    experience! If you like what you see and have a project
+                    you need coded, don’t hestiate to contact me.</p>
                 <!-- Contacts -->
-             <a class="contact" href="#contacts">LET'S CONNECT</a>
+                <a class="contact" href="#contacts">LET'S CONNECT</a>
+                <!-- Social media links -->
+                <ul class="social-media">
+                    <li><a href="https://twitter.com/jerryowusu11">
+                            <i class="fa fa-twitter fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://linkedin.com/in/jeremiahowusu">
+                            <i class="fa fa-linkedin fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://medium.com/jerryowusu">
+                            <i class="fa fa-medium fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://github.com/jerryowusu">
+                            <i class="fa fa-github fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                    <li><a href="https://angellist.com/jerryowusu">
+                            <i class="fa fa-angellist fa-lg" aria-hidden="true"></i>
+                        </a></li>
+                </ul>
+            </section>
+        </div>
+        <!-- work section -->
+        <div class="works">
+            <section class="work-card"> 
+                <img class="profile-image" src="images/tanne-chris.png" alt="portfolio snapshot" width="295"
+                    height="220">
+                <article class="client-info">
+                    <h2 class="project-title">Tonic</h2>
+                    <ul class="credentials-list">
+                        <li>CANOPY</li>
+                        <li><i class="fa fa-circle fa-sm" aria-hidden="true"></i></li>
+                        <li>Back End Dev</li>
+                        <li><i class="fa fa-circle fa-sm" aria-hidden="true"></i></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="language-list">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
```

Comment:
[OPTIONAL] 
For each project, the button (e.g., "See this project") is a `button` tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.
<img width="289" alt="image" src="https://user-images.githubusercontent.com/77190203/148660018-6d46c49c-f151-4e5f-ade6-a8b1df3ea170.png">
I strongly recommend doing that before merging your pull request